numpy/core/src/npymath/halffloat.c

```diff
@@ -281,7 +281,7 @@ npy_uint16 npy_floatbits_to_halfbits(npy_uint32 f)
     if (f_exp <= 0x38000000u) {
         /*
          * Signed zeros, subnormal floats, and floats with small
-         * exponents all convert to signed zero halfs.
```

Comment:
This is an intentional spelling, I think.

Reply:
`half-floats` is what they are.